compute/cloud-client/src/main/java/compute/disks/ResizeRegionalDisk.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.disks;
+
+// [START compute_regional_disk_resize]
+
+import com.google.cloud.compute.v1.Operation;
+import com.google.cloud.compute.v1.RegionDisksClient;
+import com.google.cloud.compute.v1.RegionDisksResizeRequest;
+import com.google.cloud.compute.v1.ResizeRegionDiskRequest;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ResizeRegionalDisk {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String projectId = "your-project-id";
+
+    // A link to the disk you want to resize.
+    // This value uses the following format:
+    //     * projects/{project_name}/regions/{region}/disks/{disk_name}
```

Comment:
Just checking if this is the only format. Does it not accept the full URL?

Reply:
The input can be in any format as we have the pattern matcher to extract the relevant portion. The format listed here is irrelevant to the API format and hence I prefer to not mention it here.